### PR TITLE
Update ZSmart Systems libraries to 1.3.0

### PR DESCRIPTION
**Pull request by cdjackson**
Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -216,13 +216,6 @@
         </executions>
       </plugin>
     </plugins>
-    <extensions>
```
Comment (kaikreuzer) [inline pom.xml]: This looks like an accidential removal?

**Comment by cdjackson** [comment]
Yep - well spotted. Not sure how that crept in.

Thanks.

**Comment by kaikreuzer** [comment]
PR build failed with a compilation failure.

**Comment by cdjackson** [comment]
I merged #548 into this PR after doing my tests, and it didn't merge the imports correctly.

**Pull request merged**